spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BoundElementFactory.java

```diff
@@ -19,26 +19,35 @@
 import org.springframework.messaging.SubscribableChannel;
 
 /**
- * Defines methods to create/configure the {@link org.springframework.messaging.MessageChannel}s defined
- * in {@link org.springframework.cloud.stream.annotation.EnableBinding}.
+ * Defines methods to create/configure the bound elements defined by
+ * {@link org.springframework.cloud.stream.annotation.EnableBinding}.
+ *
+ * @author Marius Bogoevici
  * @author Ilayaperumal Gopinathan
  */
-public interface BindableChannelFactory {
+public interface BoundElementFactory {
```

Comment:
An earlier version used 'BindingTargetFactory' so it wouldn't be hard to change back. I liked 'BoundElement' for some reason, but I'm fine with 'BindingTarget'.